test/test_ima_verification.py

```diff
@@ -281,7 +281,6 @@ def test_read_allowlist(self):
         self.assertEqual(
             al_data["meta"]["generator"], "keylime-legacy-format-upgrade", "AllowList metadata generator is correct"
         )
-        self.assertNotIn("checksum", al_data["meta"], "AllowList metadata no checksum")
```

Comment:
You removed the test that checked for a checksum, how about adding a test to verify the checksum?

Reply:
fair enough

Reply:
Just a note, checksums within allowlist objects will no longer be supported with the IMA policy overhaul (because referring to a file's own checksum within that same file will change the checksum, making it useless).